From 8dba234e2ca7c4770d0670ee7639e1e610fa9031 Mon Sep 17 00:00:00 2001
From: Stephan Gerhold <stephan@gerhold.net>
Date: Thu, 14 Oct 2021 20:55:42 +0200
Subject: [PATCH] arm64: dts: qcom: msm8916: Reduce reserved memory for venus

According to the ELF headers of the venus firmware binaries, only
~ 5 MiB (0x500000) of the venus memory region will actually end up
being used. Reduce the size of the reserved-memory region to make
slightly more RAM available for Linux.

Signed-off-by: Stephan Gerhold <stephan@gerhold.net>
---
 arch/arm64/boot/dts/qcom/msm8916.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/msm8916.dtsi b/arch/arm64/boot/dts/qcom/msm8916.dtsi
index 29ad99abbe68ed..b3a04c0c8fb1ab 100644
--- a/arch/arm64/boot/dts/qcom/msm8916.dtsi
+++ b/arch/arm64/boot/dts/qcom/msm8916.dtsi
@@ -89,7 +89,7 @@
 		};
 
 		venus_mem: venus@89900000 {
-			reg = <0x0 0x89900000 0x0 0x600000>;
+			reg = <0x0 0x89900000 0x0 0x500000>;
 			no-map;
 		};
 

